include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -1561,7 +1571,7 @@ namespace internal
                                                          values_dofs);
                   break;
                 case 1:
-                  values_dofs[2 * c] = values_quad[c][0];
+                  values_dofs[0] = values_quad[0];
```

Comment:
Should this be `values_quad[0][0]`?

Reply:
No it should not. That was a bug in itself. (I had found that while debugging when all entries got the value from the first lane of the SIMD array.)